contracts/interfaces/IRetirementCertificates.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-FileCopyrightText: 2021 Toucan Labs
+//
+// SPDX-License-Identifier: UNLICENSED
+
+// If you encounter a vulnerability or an issue, please contact <security@toucan.earth> or visit security.toucan.earth
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC721/IERC721Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/AddressUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/StringsUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+
+import "./IToucanContractRegistry.sol";
+
+interface RetirementCertificates is IERC721Upgradeable {
+    function version() external returns (string memory);
```

Comment:
Why do you need to read the `version` of the contract?

Reply:
It's a public method, maybe people could be curious or could have some sort of edge case for it. I don't really see a drawback to keeping it.

Reply:
Good question. Would be interesting in learning best practices for what to include in interfaces. 
The question is, why would another contract want to call this function? (but there are possible use cases...)

Reply:
If you guys really want that gone from the interface, I'm not against it 🤷🏻‍♂️.

Reply:
I'd be in favor of removing until proven that it's needed. It sounds like a not so obvious use case to need this function so whoever has it will most likely know how to update their interfaces. :)

Reply:
Removed it will be then